examples/xgboost_cv.py

```diff
@@ -0,0 +1,109 @@
+"""
+Optuna example that performs Cross Validation for cancer dataset
+using XGBoost.
+In this example, we perform cross-validation to accuracy of cancer detection
+using XGBoost. We optimize both the choice of booster model and their hyper
+parameters.
+We have following two ways to execute this example:
+(1) Execute this code directly.
+    $ python xgboost_cv.py
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize xgboost_cv.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+"""
+
+
+import optuna
+from optuna.samplers import TPESampler
+import os
+import sklearn.datasets
+import sklearn.metrics
+import xgboost as xgb
+
+
+# Set path to save CV results
+file_path = "./tmp_results_xgbCV_optuna/"
+
+# Create folder if it doesnt exist
+if not os.path.exists(file_path):
+    try:
+        os.makedirs(file_path)
+    except Exception:
+        print("please create folder to store xgboost CV results")
+
+# Set Constants
+seed = 108
+n_folds = 3
```

Comment:
Please use capital letters for constants.
```suggestion
SEED = 108
N_FOLDS = 3
```

Reply:
Changed